vendor/modules.txt

```diff
@@ -280,7 +280,7 @@ github.com/dustin/go-humanize
 # github.com/elazarl/goproxy v0.0.0-20180725130230-947c36da3153
 ## explicit
 github.com/elazarl/goproxy
-# github.com/emicklei/go-restful v2.9.5+incompatible
+# github.com/emicklei/go-restful v2.10.0+incompatible
```

Comment:
it is weird that random stuff that `nadtypes` doesn't even depend on got updated here... what commands did you run to update the `go.mod` stuff?